src/cirrus/management/commands/manage.py

```diff
@@ -314,3 +314,126 @@ def get_payloads(
         rerun,
     ):
         click.echo(json.dumps(payload, default=str))
+
+
+@manage.command("list-workflows")
+@pass_deployment
+def list_workflows(deployment: Deployment):
+    """List available collections/workflows combinations"""
+    workflows = deployment.get_workflows()
+    click.echo(json.dumps(workflows, indent=2))
+
+
+@manage.command("get-workflow-summary")
+@click.argument("collections")
+@click.argument("workflow_name")
+@click.option(
+    "--since",
+    default=None,
+    help=(
+        "Only include items updated since this relative duration (e.g., 7d, 36h, 15m)"
+    ),
+)
```

Comment:
It seems like it might be good to make a custom type for this. See https://click.palletsprojects.com/en/stable/parameter-types/#how-to-implement-custom-types.

Reply:
Yeah. Done.